django_ipam/tests/base/test_models.py

```diff
@@ -142,6 +142,26 @@ def test_invalid_master_subnet(self):
         if failed:
             self.fail('ValidationError not raised')
 
+    def test_correct_master_subnet_pass(self):
+        subnet1 = self._create_subnet(subnet='12.0.56.0/24')
+        try:
+            subnet2 = self._create_subnet(subnet='12.0.56.0/25', master_subnet=subnet1)
+            self._create_subnet(subnet='12.0.56.0/26', master_subnet=subnet2)
+        except ValidationError:
+            self.fail('Correct master_subnet not accepted')
+
+    def test_correct_master_subnet_fail(self):
+        failed = True
+        subnet1 = self._create_subnet(subnet='12.0.56.0/24')
+        try:
+            self._create_subnet(subnet='12.0.56.0/25', master_subnet=subnet1)
+            self._create_subnet(subnet='12.0.56.0/26', master_subnet=subnet1)
```

Comment:
which of these 2 lines is supposed to raise the `ValidationError`?  Can you keep only the line which is supposed to raise the `ValidationError` in the `ŧry/except` block?